src/lib/svg_text_utils.js

```diff
@@ -400,7 +400,12 @@ function buildSVGText(containerNode, str) {
     }
 
     function exitNode(type) {
+        // A bare closing tag can't close the root node. If we encounter this it
+        // means there's an extra closing tag that can just be ignored:
+        if(nodeStack.length === 1) return;
```

Comment:
or maybe not... not really sure, I haven't looked there. But anyway `Lib.log` points to `console.trace` if it exists, otherwise it falls back on `console.log`